external/README.md

```diff
@@ -2,7 +2,7 @@
 
 The current published artifacts are:
 
-* [Spark distribution file](https://github.com/lightbend/spark/releases/download/cloudflow-2.4.5-k8s-client-upgrade/spark-2.4.5-bin-cloudflow-2.12.tgz).
+* Spark distribution file.
```

Comment:
@yuchaoran2011 pls add the dist file at the latest  Cloudflow Spark release our fork and then I can update this PR or we can fix this link issue later.

Reply:
I don't have it now since I didn't keep the one that I generated when cutting the release, but I'll produce it today and add it as an asset to the Github release in our fork.

Reply:
The tgz is now uploaded at https://github.com/lightbend/spark/releases/download/custom-2.4.5/spark-2.4.5-bin-cloudflow-2.12.tgz.